css/style.css

```diff
@@ -0,0 +1,1085 @@
+}
+
+.ditem {
+  margin-bottom: 20px;
+}
+
+.dealtitlediv {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.imgcont {
+  position: relative;
+  overflow: hidden;
+}
+
+.ptag {
+  position: absolute;
+  top: -20px;
+  right: -110px;
+  transform: rotate(45deg);
+  background-color: #ffe712;
+  height: 25px;
+  width: 250px;
+  padding-top: 50px;
+  text-align: center;
+  font-size: 90%;
+  font-family: "Varela Round", sans-serif;
+}
+
+/* styling for footer section */
+
+footer {
+  background-color: #1a1a1a;
+  padding: 2% 0 0 0;
+  text-align: center;
+}
+
+.iconcont {
+  text-align: center;
+  margin: 0 auto;
+}
+
+.ficon {
+  color: white;
+  margin: 10px;
+  font-size: 100%;
+  background-color: #333;
+  padding: 8px;
+  border-radius: 50%;
+  border: 2px solid #333;
+}
+
+.ficon:hover {
+  background-color: black;
+  border: 2px solid grey;
+}
+
+.footnav {
+  text-align: center;
+  width: 25%;
+  margin: 2% auto;
+  font-size: 75%;
+  padding-bottom: 20px;
+}
+
+.footnav li {
+  list-style-type: none;
+  float: left;
+}
+
+.footnav a:hover {
+  color: white;
+}
+
+.footbottom {
+  clear: both;
+  margin: 0;
+  background-color: #000;
+  padding: 2% 0;
+  text-align: center;
+}
+
+/* responsiveness for medium devices */
+@media only screen and (max-width: 1023px) and (min-width: 768px) {
+  /* top navigation styling */
+  .navtop {
+    background-color: black;
+    padding: 10px;
+    font-size: 80%;
+    margin: 0;
+    height: 30px;
+  }
+
+  /* navlogom&s style */
+  .navlogom-m {
+    background: linear-gradient(to bottom right, #c09, #f42 85%);
+    width: 106px;
+    height: 60px;
+    text-align: center;
+    padding-top: 30px;
+    display: block;
+    position: absolute;
+    top: 0;
+    z-index: 200;
+  }
+
+  .navlist {
+    margin-left: 80px;
+  }
+
+  .navlogom-m img {
+    width: 70%;
+  }
+
+  .navlogom-s {
+    display: block;
+  }
+
+  .navlogom-s img {
+    width: 70%;
+  }
+
+  /* navigation fixed styling */
+  .header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 100;
+  }
+
+  .navfixed {
+    height: 40px;
+    font-family: "Nova Square", cursive;
+    background-color: white;
+    font-size: 90%;
+  }
+
+  .navli2 {
+    margin-left: -60px;
+  }
+
+  #searchbtn {
+    border-bottom: 1px rgba(84, 84, 84, 0.27) solid;
+    padding-bottom: 10px;
+    width: 25px;
+    padding-left: 15px;
+    font-size: 110%;
+  }
+
+  .hiddenmd {
+    display: none;
+  }
+
+  /* yellow nav styling */
+
+  .yellow-nav {
+    background-color: #ffe712;
+    height: 20px;
+    padding: 5px;
+    text-align: center;
+    clear: both;
+  }
+
+  /* main content */
+  .main-news-section {
+    display: grid;
+    grid-gap: 6px;
+    grid-template-areas:
+      "main side-top "
+      "main side-bottom";
+    height: 50vh;
+    margin: 4px 4px 5% 4px;
+  }
+
+  .main-pic {
+    /* background:  linear-gradient(rgba(196, 32, 32, 0.25)),url("./images/main-pic-big.png"); */
```

Comment:
Please remove this commented piece of code.

Reply:
Commented code removed.